webview/platforms/winforms.py

```diff
@@ -547,6 +549,57 @@ def alert(message):
         WinForms.MessageBox.Show(str(message))
 
 
+class OpenFolderDialog:
+    foldersFilter = "Folders|\n"
+    flags = BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic
+    windowsFormsAssembly = Assembly.LoadWithPartialName("System.Windows.Forms")
+    iFileDialogType = windowsFormsAssembly.GetType("System.Windows.Forms.FileDialogNative+IFileDialog")
+    OpenFileDialogType = windowsFormsAssembly.GetType("System.Windows.Forms.OpenFileDialog")
+    FileDialogType = windowsFormsAssembly.GetType("System.Windows.Forms.FileDialog")
+    createVistaDialogMethodInfo = OpenFileDialogType.GetMethod("CreateVistaDialog", flags)
+    onBeforeVistaDialogMethodInfo = OpenFileDialogType.GetMethod("OnBeforeVistaDialog", flags)
+    getOptionsMethodInfo = FileDialogType.GetMethod("GetOptions", flags)
+    setOptionsMethodInfo = iFileDialogType.GetMethod("SetOptions", flags)
+    fosPickFoldersBitFlag = windowsFormsAssembly.GetType(
+        "System.Windows.Forms.FileDialogNative+FOS").GetField("FOS_PICKFOLDERS").GetValue(None)
+
+    vistaDialogEventsConstructorInfo = windowsFormsAssembly.GetType(
+        "System.Windows.Forms.FileDialog+VistaDialogEvents").GetConstructor(flags, None, [FileDialogType], [])
+    adviseMethodInfo = iFileDialogType.GetMethod("Advise")
+    unadviseMethodInfo = iFileDialogType.GetMethod("Unadvise")
+    showMethodInfo = iFileDialogType.GetMethod("Show")
```

Comment:
Please change double quotes to single ones

Reply:
> Please change double quotes to single ones

OK